### ISSUE TITLE
".switch input" class display:none cause accessibility issue

### ISSUE THREAD

**Issue by shadow2560** [Infarctus/CO-E33_Save_editor#8]
**Describe the bug**
In the css you have:
|```
.switch input {
  display: none;
}
```
The problem is that this part of the code this not display the checkbox in the function renderToggle() (logic cause the label use the class "switch") but this cause a problem with screen readers witch can't see the checkbox and can't interact with them. I've tried to replace "display: none;" with "display: block;" in the css and with that it OK for accessibility but I don't know what are the visual effects of this modification (you probably haven't do this for no reason) so I prefer to not propose a pull request with this modification and let you define a better approach to fix this issue.
Note: I've also tried to let "display: none;" and add "aria-hidden: false;" and "aria-disabled: false;" into the css ".switch input" but doesn't work, checkbox are always not accessibles.

**To Reproduce**
Steps to reproduce the behavior:
1. Launch a screen reader like NVDA
2. Open a save
3. Go to 'pictos' or any category that display a checkbox to enable or disable a state, except for kill/unkill white nevrons witch work properly
4. Scroll down to a checkbox in the table using the keyboard
5. Checkbox are not announced and can't have keyboard interactions

**Expected behavior**
Should announce the checkbox and interact with it

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Version: Actual master of the project, compiled by myself

**Additional context**
NVDA screen reader

**Comment by shadow2560** [comment]
Work with this and should not interfere with visual:
```
.switch input {
  position: absolute;
  opacity: 0;
  width: 0;
  height: 0;
}
```

But I can't confirm it visualy and I don't know if this could brake other elements so I prefer to let you choose if it's the correct solution to fix this.

**Comment by Infarctus** [comment]
Hello, first of all thank you for your research on it, we'll be seriously looking into this to try to fix the problem, as long as there's no "display:none" there should be no issue with the screen reader ?

**Comment by shadow2560** [comment]
Yes, using "display: none" hide the element visually and also hide the element on accessibility tree so impossible to make something accessible with this property set to "none" except by using an other accessible element witch can interact with it (in this case totally unuseful  to do that cause you hide an element to replace it with an other visible element witch will cause the same problems).

And just a question, in the menu "Unkill Enemies" all checkbox are checked (I haven't kill all ennemies listed in my save), is it normal? I suppose it's normal and we just need to uncheck the checkbox to validate the unkill but I prefer to ask.

**Comment by Infarctus** [comment]
Hello, yes it only lists enemies you have killed on your saves and only them, not every single enemy that you can encounter. ^^

**Comment by Infarctus** [comment]
Btw just out of curiosity, how do you play the game ?

**Comment by shadow2560** [comment]
Thanks for fixing the problem.

To answer your question it's really hard to explain, I use the little vision that I always have, the sounds of the game and OCR to read texts (pictos description, skills, tutorials, etc...). I also use a free camera mod to teleport some times and to save/restore my position to help me in some zones. The most difficult parts for me is the exploration and preparation, not the fights. But even with that some things could probably be improved to help, I've opened a subject to explain the difficulties and how they can be resolved, see here:
https://forums.nexusmods.com/topic/13517182-clair-obscur-expedition-33-accessibility-improvements/

I've tried to learn how to dev mods but all the tools are totally inaccessible with screen readers (Unreal Engine, UE4SS gui console, etc...) so I can't, for example, try to find a changed value during the game to implement a new function. I've also tried to dev a real time dump mod witch write to a log file (I can't read results in the game's console) but without success for now, I can write into the file but I don't understand how to dump what I want for now.
